### PR TITLE
[plugin-parser] Add plugin-parser to handle rpk packages

### PR DESCRIPTION
**Pull request by anyj0527**
- Remove pkg-mgr module from the daemon
- Add libmlops-plugin-parser.so to handle package install / upgrade / uninstall for Tizen

### CODE REVIEW

File: plugin-parser/mlops-plugin-parser.cc [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+  list = g_list_first (metadata);
+  while (list) {
+    detail = (_metadata_s *) list->data;
+    _I ("key = %s, value = %s\n", detail->key, detail->value);
+    list = g_list_next (list);
+  }
+  pkgmgrinfo_pkginfo_destroy_pkginfo (handle);
+
+  g_autoptr (GError) err = NULL;
```
Comment (jaeyun-jung) [inline plugin-parser/mlops-plugin-parser.cc:102]: this is unnecessary

File: packaging/mlops-agent.spec [hunk 2]
```diff
@@ -242,6 +242,8 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
 %attr(0644,root,root) %{_unitdir}/mlops-agent.service
 %attr(0644,root,root) %config %{_sysconfdir}/dbus-1/system.d/mlops-agent.conf
 %attr(0644,root,root) %{_datadir}/dbus-1/system-services/org.tizen.machinelearning.service.service
+%{_sysconfdir}/package-manager/parserlib/metadata/libmlops-plugin-parser.so
```
Comment (jaeyun-jung) [inline packaging/mlops-agent.spec:245]: need checking package to install parser library
mlops-agent or libmlops-agent

Comment (jaeyun-jung) [reply packaging/mlops-agent.spec:245]: With offline discussion, add plugin library here. :)

**Comment by anyj0527** [comment]
Consider the cpp-linter is now broken https://github.com/cpp-linter/cpp-linter-action/issues/214

File: plugin-parser/mlops-plugin-parser.cc [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+        break;
+      default:
+        _E ("Unknown data type '%d', internal error?", json_type);
+        break;
+    }
+  }
+}
+
+extern "C" int
+PKGMGR_MDPARSER_PLUGIN_INSTALL (const char *pkgid, const char *appid, GList *metadata)
```
Comment (jaeyun-jung) [inline plugin-parser/mlops-plugin-parser.cc:221]: IMO need '@brief' tag for all functions.

Comment (anyj0527) [reply plugin-parser/mlops-plugin-parser.cc:242]: I added those tags. PTAL

**Review by jaeyun-jung — APPROVED** [review]

**Comment by jaeyun-jung** [comment]
Ignore lint error, merge this PR.

**Comment by jaeyun-jung** [comment]
@yeonykim2 please update json format on this parser.

**Pull request merged**